simple_salesforce/login.py

```diff
@@ -195,8 +196,7 @@ def SalesforceLogin(
                 )
             }
         if privatekey_file is not None:
-            with open(privatekey_file, 'rb') as key_file:
-                key = key_file.read()
+            key = Path(privatekey_file).read_bytes()
```

Comment:
I am not as familiar with pathlib but only concern is if this would prevent the use of s3 or other cloud storage buckets

Reply:
There isn't any code re: remote file paths in this package. It was just one of the main things I saw when quickly looking into the differences between the two but digging a bit deeper it is not applicable here as it would be handled outside of this package.